lib/components/search_user/interactor/search_user_bloc.dart

```diff
@@ -43,7 +43,13 @@ class SearchUserBloc extends Bloc<SearchUserEvent, SearchUserState> {
       if (event.searchQuery.length > _minTextLengthBeforeValidSearch) {
         yield state.copyWith(pageState: PageState.loading);
         final results = await SearchForMemberUseCase().run(event.searchQuery);
-        yield SearchUserStateMapper().mapResultToState(state, results[0], results[1], state.noShowUsers);
+        yield SearchUserStateMapper().mapResultToState(
+          currentState: state,
+          seedsMembersResult: results[0],
```

Comment:
i know this is how it was done before. 
But why not pass in results

Then inside the mapper you can move the logic results[0], results[1], etc

Reply:
Really want them named - e.g. I really want to know which is which. We could pass an object that has named lists. Wouldn't want to have to remember which index is what.

Reply:
you either remember which index is what here or Inside the mapper. I would argue that its the mapper's job to remember and not the block.

Reply:
im not blocking this PR on this btw, its just a suggestion .

Reply:
Yeah I see what you mean

Reply:
Really whatever is making the network calls should keep track of the lists and name them